backend/open_webui/utils/utils.py

```diff
@@ -75,10 +73,15 @@ def get_http_authorization_cred(auth_header: str):
     except Exception:
         raise ValueError(ERROR_MESSAGES.INVALID_TOKEN)
 
+def get_api_key_auth_config():
```

Comment:
If we expect the option to not be a `ConfigurableOption` & that the value is only upon establishing the environment, then we should also opt to use `@lru_cache` decorator.